inc/languages/english/admin/forum_attachments.lang.php

```diff
@@ -78,7 +78,7 @@
 $l['index_find_attachments'] = "Attachments - Find Attachments";
 $l['find_where'] = "Find attachments where&hellip;";
 $l['name_contains'] = "File name contains";
-$l['name_contains_desc'] = "To search enter .[file extension]. Example: .zip.";
+$l['name_contains_desc'] = "To search by file extension, type <strong>.[file extension]</strong>. Example: <strong>.zip</strong>.";
```

Comment:
I don't think HTML should be added to the language files after the large effort to move all HTML to templates: #756 #2025

Reply:
I'd agree. I wonder if `To search by file extension, type '.[file extension]'. Example: '.zip'.` would be clear enough?

Reply:
> I don't think HTML should be added to the language files after the large effort to move all HTML to templates: #756 #2025

IMO it's kind of based on the web page designer / language pack author. I personally tend to use `<strong>`/`<em>` for emphasizing.

> I'd agree. I wonder if `To search by file extension, type '.[file extension]'. Example: '.zip'.` would be clear enough?

I'm not sure because someone would probably type exactly `'.[file extension]'`/`'.zip'` into the input area. And it's a little bit common to see some page say `please type "<something_should_be_entered>" (without quotes) in the text box`.

Anyway, it's OK for me to have it written in either way and let it be in MyBB's way.

Reply:
Perhaps just rewriting so the `.zip` doesn't fall right before a full stop will avoid confusion enough?

```suggestion
$l['name_contains_desc'] = "Search for attachments that include the given query in the file name. For example, enter .zip to find attachments using the .zip file extension.";
```

Reply:
> Perhaps just rewriting so the `.zip` doesn't fall right before a full stop will avoid confusion enough?

Looks good to me. And that's what @Eldenroot was calling. 🤣